dev-docs/transformation-tests.md

```diff
@@ -0,0 +1,43 @@
+# Transformation Tests
+
+The transformation tests are tests that will run Fluent Bit and OpenTelemetry Operations Collector (otelopscol) directly in your environment and test the results of various logging operations to stdout. This is to check the correctness of log parsing expectations in a more lightweight way than full integration tests.
```

Comment:
"test… to stdout" sounds a bit off. Did you mean "test… *via* stdout" (i.e., "…by printing to stdout")?